src/main/resources/sql/db_sql.sql

```diff
@@ -0,0 +1,32 @@
+drop table vesta_schema.user;
+drop table vesta_schema.user_role;
+drop table vesta_schema.role;
+
+create table vesta_schema.user(
+    id serial primary key,
+    name varchar (50) UNIQUE not null ,
+    lastname varchar (50) unique  not null ,
```

Comment:
lastname dosen't need to be unique and by naming convention if field is composed from two or more words they are separated by _ ex: lastname -> last_name